.github/workflows/tests.yml

```diff
@@ -0,0 +1,65 @@
+name: CI
+
+on:
+  push:
+    branches: master
+  pull_request:
+    branches: master
+  schedule:
+    # * is a special character in YAML so you have to quote this string
+    - cron:  '0 3 * * 6'
+  workflow_dispatch:
+    inputs:
+      reason:
+        description: 'Reason'
+        required: false
+        default: 'Manual trigger'
+
+jobs:
+  Tests:
+    runs-on: ubuntu-20.04
+    strategy:
+      matrix:
+          python-version: [3.6, 3.7, 3.8]
+          services: [lowest, release, devel]
```

Comment:
Shall we follow the same naming as https://github.com/inveniosoftware/invenio-records/blob/897dcb3b8e212acee8860a6fd44789434174c00e/.github/workflows/tests.yml#L23-L25 ?